experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/fml.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Diagnostic } from "@codemirror/lint";
+import {
+  createDiagnostic,
+  documentPosition,
+  parseError,
+  TestableEditorView,
+} from "src/components/PageEditBranches/FormBranches/FormFeatureValue/helpers";
+import { getConfig_nimbusConfig_applicationConfigs_channels } from "src/types/getConfig";
+import { getExperiment_experimentBySlug_featureConfigs } from "src/types/getExperiment";
+
+const line = 1;
+const column = 2;
+
+/**
+ * Parse from FmlErrors to Diagnostic
+ */
+function parseFmlErrors(
+  view: TestableEditorView,
+  fmlErrors: unknown[],
+): Diagnostic[] {
+  const diagnostics: Diagnostic[] = [];
+
+  for (const error of fmlErrors) {
+    const err = parseError.parse(error);
+    const pos = documentPosition(view.state.doc, err.line, err.column - 1);
+
+    diagnostics.push(createDiagnostic(pos, pos, err.rawMessage));
```

Comment:
Nit: if `highlight` exists, then we should add the length of it to the starting `pos` to get the end.

e.g. 
```suggestion
    diagnostics.push(createDiagnostic(pos, pos + err.highlight?.length ?? 1, err.rawMessage));
```